src/backend/sgx/ioctls.rs

```diff
@@ -23,18 +23,21 @@ pub const ENCLAVE_ADD_PAGES: Ioctl<WriteRead, &AddPages<'_>> = unsafe { SGX.writ
 /// IOCTL identifier for EINIT (see Section 41-35)
 pub const ENCLAVE_INIT: Ioctl<Write, &Init<'_>> = unsafe { SGX.write(0x02) };
 
-pub const ENCLAVE_SET_ATTRIBUTE: Ioctl<Write, &SetAttribute<'_>> = unsafe { SGX.write(0x03) };
+/// SGX_IOC_ENCLAVE_SET_ATTRIBUTE (0x03) is not used by Enarx.
 
 /// SGX_IOC_VEPC_REMOVE_ALL (0x04) is not used by Enarx.
 
-/// SGX_IOC_ENCLAVE_RELAX_PERMISSIONS
-pub const ENCLAVE_RELAX_PERMISSIONS: Ioctl<WriteRead, &RelaxPermissions<'_>> =
-    unsafe { SGX.write_read(0x05) };
-
 /// SGX_IOC_ENCLAVE_RESTRICT_PERMISSIONS
 pub const ENCLAVE_RESTRICT_PERMISSIONS: Ioctl<WriteRead, &RestrictPermissions<'_>> =
```

Comment:
The reference above actually is "a bit broad" since it's few hundred pages and contains also these magic numbers :-)

Reply:
I'll remove that reference because it tells **nothing** and replace more scoped xref's.

Reply:
Ugh, ignore my previous comments, was mixing ioctl's and SDM.

Reply:
BTW, those references are bit weird as these opcodes do not have named sections, and using e.g. 41-21 as keyword does not lead you to ECREATE in that PDF. So they probably change randomly in each release of the SDM.

It would be better idea to have a single conclusive comment:

```rust
/// The ioctl's provide an interface to the privileged opcode ENCLS,
/// which is used to manage enclaves. The different sub-functions
/// are documented in Intel SDM Volume 3D, section 37.3.
```